### PR TITLE
The Ember Times No. 144 - April 17th 2020

### PR DESCRIPTION
**Pull request by amyrlam**
📝 Ideas, feel free to add to list or claim!

- [x] Virtual EmberConf workshops available to everyone (🔒 @jaredgalanis )
  - https://twitter.com/ember_map/status/1245011062417772548
  - https://twitter.com/ember_map/status/1246071188394250240
  - https://github.com/simplabs/ast-workshop
- [x] Quest Issue to Drop Mixins for ember-simple-auth (https://twitter.com/marcoow/status/1250411978339819522) (🔒 @chrisrng )
- [x] Updated documentation for ember-statecharts
  - https://twitter.com/LevelbossMike/status/1248247002489425921 (🔏 @jayjayjpg) 
- [ ] https://medium.com/@mudflye/undo-and-redo-with-orbit-js-ember-example-8df226b6ea52
- [x] Writeup on the work of the a11y strike team + call for contributors for a11y related accessibility issues. Ping @MelSumner for more context and writeup review (🔏 @jayjayjpg)
- [x] FCP https://twitter.com/emberjs/status/1251205640296509440
https://twitter.com/emberjs/status/1251238742393794561 from locks 

📆 Future stories

- [ ] Simple Drag and Drop in Ember Octane (:lock: @roderickdevelopers)
  - https://twitter.com/dgavey/status/1248070978329960448

🤔 Questions?

See #support-ember-times on Discord or CONTRIBUTING.

✅ Todos:

- Check https://github.com/ember-learn/whats-new-in-emberland (roundup of all the PRs in emberjs and ember-learn repos) for ideas
- Use https://github.com/ember-learn/emberjs-times-tools README
- Check Netlify preview, to ensure display on emberjs.com/blog is good
  - [ ] Check buttons
- [ ] Crosspost on dev.to, delete Readers Question picture, add Canonical URL 
- [ ] Set up tweet for Monday morning on tweetdeck
  - [ ] post $natural

### CODE REVIEW

**Comment by amyrlam** [comment]
cc @abhilashlr @sharpshark28 if you happen to be interested in writing or editing this week, no worries if not!

File: source/2020-04-17-the-ember-times-issue-144.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+---
+
+## [Call for contributors: Join the strike team for accessibility in Ember](https://github.com/emberjs/rfcs/issues/595)
+
+The Ember community put in lots of work into [framework features](https://guides.emberjs.com/release/accessibility/) and [addons](https://emberobserver.com/categories/accessibility) to allow us to build accessible and user-friendly applications. But still there is more work to be done to **address accessibility issues** in Ember.js and to improve the future of how we build JavaScript apps.
+
+Do you want to be part of these efforts to help modern Ember apps become accessible?
+
+Then be sure join the **#st-a11y strike team** on the [Ember Discord chat](https://discordapp.com/invite/emberjs)! The recently founded strike team for creating an **accessible JavaScript framework** has already put together a [detailed list of issues to work on](https://github.com/emberjs/rfcs/issues/595) and is counting on your support. If you have the time and interest to help out, [join the team today](https://discordapp.com/invite/emberjs)!
```
Comment (MelSumner) [inline source/2020-04-17-the-ember-times-issue-144.md]:
```suggestion
Then be sure join the **#st-a11y strike team** on the [Ember Discord chat](https://discordapp.com/invite/emberjs)! The recently founded strike team for creating an **accessible JavaScript framework** has already put together a [detailed list of issues to work on](https://github.com/emberjs/rfcs/issues/595) and is counting on your support. If you have the time and interest to help out, [join the team today](https://discord.gg/emberjs)!
```

File: source/2020-04-17-the-ember-times-issue-144.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+Call for a11y-focussed contributors ❤️,
+READMORE
+
+---
+
+## [Call for contributors: Join the strike team for accessibility in Ember](https://github.com/emberjs/rfcs/issues/595)
+
+The Ember community put in lots of work into [framework features](https://guides.emberjs.com/release/accessibility/) and [addons](https://emberobserver.com/categories/accessibility) to allow us to build accessible and user-friendly applications. But still there is more work to be done to **address accessibility issues** in Ember.js and to improve the future of how we build JavaScript apps.
+
+Do you want to be part of these efforts to help modern Ember apps become accessible?
```
Comment (MelSumner) [inline source/2020-04-17-the-ember-times-issue-144.md]:
```suggestion
Do you want to be part of these efforts to help modern Ember apps become more accessible?
```

File: source/2020-04-17-the-ember-times-issue-144.md (hunk 1 above)
Comment (amyrlam) [reply source/2020-04-17-the-ember-times-issue-144.md]: just noticed this, was out of date, so manually made the change myself

**Pull request merged**